actionpack/test/dispatch/system_testing/system_test_case_test.rb

```diff
@@ -5,33 +5,41 @@
 
 class SetDriverToRackTestTest < DrivenByRackTest
   test "uses rack_test" do
-    assert_equal :rack_test, Capybara.current_driver
+    assert_equal :rack_test, self.class.driver.instance_variable_get(:@driver_type)
```

Comment:
These test changes on `current_driver` feel incorrect to me. `Capbyara.current_driver` isn't an API we control - it should still return the same symbol it was returning before. If we also want to test that `driver_type` returns `:rack_test` then that should be an additional assertion rather than replacing this assertion.